emission/net/api/cfc_webapp.py

```diff
@@ -379,6 +382,41 @@ def getCarbonCompare():
     logging.debug("No overriding client result for user %s, returning choice" % user_uuid)
   return choice.getResult(user_uuid)
 
+
+#Pulling public data from the server  
+@get('/getData')
```

Comment:
Also, this can be fixed later, but I would suggest a more descriptive name than `getData`. Think of this like the habitica API - if you see something that says `getData`, you are not sure whether it returns the timeline or the metrics or what...

I would suggest something more descriptive - maybe `/eval/publicData/timeseries`? I think that I am too close to the project to come up with a great name but maybe you can do something better...